packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -73,6 +74,48 @@ import 'ticker_provider.dart';
 /// }
 /// ```
 /// {@end-tool}
+///
+/// Instead of switching to [NestedScrollView], the position where the glow is
```

Comment:
WDYT of putting your sample first before the NestedScrollView sample?

Reply:
Done.